README.md

```diff
@@ -12,3 +12,82 @@
+|-----------------------------|----------------------------------------------------------|
+| `make build`                | Build the application                                    |
+| `make ci-code-format`       | Check the code format according to the rust format rules |
+| `make ci-code-coverage`     | Launch tests and returns code coverage for tests         |
+| `make ci-lint`              | Launch the rust linter                                   |
+| `make ci-tests`             | Launch tests                                             |
+| `make docker-build`         | Build locally a docker image                             |
+| `make docker-clean`         | Remove locally the built docker image                    |
+| `make docker-push-registry` | Push the locally built docker image                      |
+| `make run`                  | Run locally the worker                                   |
+| `make version`              | Display the version defined in `cargo.toml` file         |
+
+## CI / CD
+
+A `.gitlab-ci.yml` file is provided for the gitlab CI/CD feature.
+This file will instantiate te following pipeline:
+
+<!-- language: lang-none -->
+    /-----------\      /---------\             /----------\             /----------\
+    |  Compile  |------|  Tests  |-------------| Quality  |-------------|  Docker  |
+    \-----------/      \---------/             \----------/             \----------/
+         |                  |                       |                        |
+     +------+           +-------+              +---------------+          +-------+
+     | lint |           | tests |              | code-coverage |          | build |
+     +------+           +-------+              +---------------+          +-------+
+                                                    |
+                                               +-------------+
+                                               | code-format |
+                                               +-------------+
+<!-- language: markdown -->
+
+### Docker
+
+The command `make docker-build` will build an image named `mediacloudai/rdf_worker`.
+
+The command `make push-docker-registry` will logged in and push the built image in the official docker registry. The login must be set with the following environment variables:
+
+| Variable name           | Default value              | Description                                                  |
+|-------------------------|----------------------------|--------------------------------------------------------------|
+| `DOCKER_REGISTRY_LOGIN` |                            | User name used to connect to the docker registry.            |
+| `DOCKER_REGISTRY_PWD`   |                            | Password used to connect to the docker registry.             |
+| `RUSTTOOLS_DOCKER_IMG`  |                            | Name of RUST image containing CLIPPY, TARPAULIN and RUSTFMT. |
```

Comment:
can we use RUST_TOOLS_DOCKER_IMG var name ?

Reply:
Yes, this variable is used in Gitlab-CI. So I moved the description is the corresponding section of the README.